ingest/monitor.py

```diff
@@ -74,3 +77,53 @@ def trace_fn(*args, **kwargs):
             return fn(*args, **kwargs)
 
     return trace_fn
+
+
+def before_send_to_sentry(event, hint):
+    """Edit event properties before logging to Sentry
+
+    Docs: https://docs.sentry.io/error-reporting/configuration/filtering/?platform=python#before-send
+    """
+    # Report a general logger name (`ingest_pipeline`) to Sentry,
+    # rather than the function-specific logger name (e.g. `__main___errors`)
+    # used internally within Ingest Pipeline.
+    event['logger'] = 'ingest_pipeline'
+    return event
+
+
+def integrate_sentry():
+    '''Log Ingest Pipeline errors to Sentry, by integrating with Python logger
+
+    General docs:
+    https://docs.sentry.io/platforms/python/logging/
+
+    API docs:
+    https://getsentry.github.io/sentry-python/api.html
+    https://getsentry.github.io/sentry-python/integrations.html
+
+    See logs in Sentry:
+    https://sentry.io/organizations/broad-institute/issues/?project=1424198
+    '''
```

Comment:
I love these links to the docs.  I might move them up to where you import sentry, since these are more general